### ISSUE TITLE
Dueling Banjo music doesn't stop after a loadout preview

### ISSUE THREAD

**Issue by Malfunktion-issues** [mastercomfig/team-comtress-2#552]
### Description

Whenever you examine the Dueling Banjo and let the music play, it does not end unless you leave the server or quit the game. Snd_stop probably works but that also kills the sound from everything else.
### Checklist

- [x] None of the open *or* closed issues document this problem.
- [x] This is a bug, not a new idea.
- [x] This is objectively a bug, and not something that people commonly disagree on.
- [untested] This issue occurs on the latest Team Comtress release, or the `master` branch.
- [x] This issue occurs on the latest vanilla TF2 update.

- [x] This bug is in the code of the game, not in the textures, sounds, or other resources.
- [ ] This issue is listed on the [Source 1 official bugtracker](https://github.com/ValveSoftware/Source-1-Games/issues).
- [x] This issue occurs with DirectX level 100.
- [x] This issue occurs when using no custom content.
- [x] This issue occurs when using `start_tf2.bat` (or `start_server.bat`) with no custom launch options.
- [x] This issue occurs when using a listen server (using the `map` command ingame).
- [x] This issue occurs when using a dedicated server.
- [ ] This bug is in the internal procedures or tooling (e.g. the build process).

### Reproduction

1. Go to Engineer's taunts in the loadout menu
2. Equip the Dueling Banjo
3. yeehaw eternally

#### Client Settings

- *Server or client:* 
- *Operating system:* 
- *Team Comtress version:* 
- *Custom content:* 
- *Launch options:* 
#### Driver Version

- *Graphics driver:* 
- *DirectX level:* 
#### Hardware Specs

- *CPU model:* 
- *GPU model:* 
- *RAM amount:* 
- *TF2 install location:* 
- *TC2 install location:* 
- *TC2 installation method:*

**Comment by AzureAzel** [comment]
This also occurs with the Scorcher's Solo.

**Comment by BigDave69** [comment]
yes i noticed this bug and it's happened every time I look at the taunt in the mann co store thank you for bringing light to this
